.github/workflows/ci.yml

```diff
@@ -0,0 +1,115 @@
+name: Nikola CI
+on:
+  # Run on all pushes to master (including merge of PR)
+  push:
+    branches: master
+  # Run on all changes in pull requests
+  pull_request:
+
+jobs:
+  nikola:
+    name: Nikola test suite (Python ${{ matrix.python }} on ${{ matrix.image }})
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [3.5, 3.6, 3.7, 3.8, 3.9]
+        image:
+          - ubuntu-latest
+        include:
```

Comment:
Neat, I haven't seen that in use before, it makes adding special cases a lot easier (and nicer)!